docs/cli/argo_server.md

```diff
@@ -29,6 +29,7 @@ See https://argoproj.github.io/argo-workflows/argo-server.md
       --event-worker-count int               how many event workers to run (default 4)
   -h, --help                                 help for server
       --hsts                                 Whether or not we should add a HTTP Secure Transport Security header. This only has effect if secure is enabled. (default true)
+      --json-log-format                      Whether to use JSON formatter for logs. Default to use plain text formatter
```

Comment:
I'd prefer `--log-format=json`

Reply:
does Argo CD do something similar at all?

Reply:
Did a quick search - looks like it does https://github.com/argoproj/argo-cd/blob/a96b476f167b08ee900887b7f9e026a06c45e899/util/cli/cli.go#L145-L156

It uses `--logformat` [here](https://github.com/argoproj/argo-cd/blob/7af5837ffabd5420b52b7980baeab19f046f9294/cmd/argocd-server/commands/argocd_server.go#L144). I think we should do `--log-format` with `text` being the default here.